docs/widgets/iframe/index.mdx

```diff
@@ -60,3 +60,12 @@ Check out our documentation on [editing tiles](/docs/getting-started/after-the-i
 
 ## Screenshots
 ![](./widget.webp)
+
+---
+
+## homarr-iframes
```

Comment:
Add an info using this tool  https://docusaurus.io/docs/markdown-features/admonitions and mention that this is third party content and all support requests / problems must be redirected to the appropriate repository.